tests/cephadm/test_cephadm_change_public_network.py

```diff
@@ -0,0 +1,25 @@
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import CephadmOpsExecutionError
+
+
+def run(ceph_cluster, **kwargs):
+    """Change public network and check cluster health"""
+
+    # Get config
+    config = kwargs.get("config")
+
+    # Get public network
+    public_network = config.get("public_network")
+
+    # Get installer node
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+
+    # Set public network
+    CephAdm(installer).ceph.config.set(key="mon public_network", value=public_network)
+
+    # Get cluster health
+    out = CephAdm(installer).ceph.health(detail=True)
```

Comment:
Only health check validation is sufficient? Don't need to check any config ?

Reply:
Hi @vamahaja 

It is sufficient

Same validation performed in polarion TC(CEPH-83575381)